index.html

```diff
@@ -166,5 +166,33 @@ <h3>LET'S CONNECT</h3>
                 <br>
             </ul>
 </section>
-</body>
-</html>
\ No newline at end of file
+<footer>
+    <form action="https://formspree.io/f/xrgdkyvk" method="post">
+        <header>
+          <legend>Contact me</legend>
+          <p>
+            If you have an application you <br>
+            are interested in developing, a <br>
+            feature that you need built or a <br>
+            project that needs coding. I’d <br>
+            to help with it
+          </p>
+        </header>
+        <ul style="list-style: none">
+          <li>
+            <input type="text" name="user-name" id="name" minlength="5" placeholder="Yeremias" required maxlength="30">
+          </li>
+          <li>
+            <input type="email" name="email" id="email" placeholder="natajayanj14@gmail.com" required>
+          </li>
+          <li>
+            <textarea name="user_message" id="msg" placeholder="Write your message here" maxlength="500" required></textarea>
+          </li>
+          <li>
+            <input type="submit" name="submit" value="Get in touch">
+          </li>
```

Comment:
- [ ] [OPTIONAL] it would be better if you remove the black border from the inputs and also add the right padding to them in order to match The Figma design 
- Yours 
<img width="359" alt="Screen Shot 2022-10-04 at 17 32 12" src="https://user-images.githubusercontent.com/61790587/193875203-c6479052-d5ad-4196-9264-d5a517e7ad69.png">